core/src/main/kotlin/org/evomaster/core/search/gene/LongGene.kt

```diff
@@ -15,23 +15,37 @@ import org.slf4j.LoggerFactory
 
 class LongGene(
         name: String,
-        value: Long = 0
+        value: Long = 0,
+        val min : Long? = null,
+        val max : Long? = null
 ) : NumberGene<Long>(name, value) {
 
+    init {
+        if ((min != null).xor(max != null)){
+            throw IllegalArgumentException("min and max should be specified together, i.e., both are null or not")
+        }
```

Comment:
I added this since `min` and `max` are only used for unsigned int4. now I remove the constraint and add a check whether the min/max is specified. if it does, we will employ the same strategy with `int` to randomize `long`.